wazimap_ng/points/tasks.py

```diff
@@ -35,6 +35,7 @@ def process_uploaded_file(point_file, subtheme, **kwargs):
         header=None,
         keep_default_na=False,
         encoding=encoding,
+        dtype=str
```

Comment:
@michaelglenister Do we need this change as well ?

Reply:
No, this is related to a different issue, I must have branched from the wrong point. I've cleaned it up